client/tokenized-payment-request/transformers/wc-to-stripe.js

```diff
@@ -3,6 +3,21 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * GooglePay/ApplePay expect the prices to be formatted in cents.
+ * But WooCommerce has a setting to define the number of decimals for amounts.
+ * Using this function to ensure the prices provided to GooglePay/ApplePay
+ * are always provided accurately, regardless of the number of decimals.
+ *
+ * @param {number} price the price to format.
+ * @param {{currency_minor_unit: {number}}} priceObject the price object returned by the Store API
+ *
+ * @return {number} the price amount for GooglePay/ApplePay, always expressed in cents.
+ */
+export const transformPrice = ( price, priceObject ) =>
+	// making sure the decimals are always correctly represented for GooglePay/ApplePay, since they don't allow us to specify the decimals.
+	price * 10 ** ( 2 - priceObject.currency_minor_unit );
```

Comment:
This is similar to this formatting function: https://github.com/woocommerce/woocommerce/blob/e5fafb455eded795d66ae0f75b2cc8a3a36320ca/plugins/woocommerce-blocks/assets/js/extensions/google-analytics/utils.ts#L59

instead of `({price} / 10^{currency_minor_unit}) * 100`, it's simplified to `{price} * 10 ^ (2 - {currency_minor_unit})`, because `100`==`10^2`.